scripts/bin/acl-config.py

```diff
@@ -0,0 +1,449 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2015 Dell Inc.
```

Comment:
Copyright update?

Reply:
ignoring.. will deal with this with a script to change all of it and put it in a separate commit.